pkg/newlog/cmd/newlogd.go

```diff
@@ -93,10 +94,12 @@ var (
 	appStatsMap map[string]statsLogFile
 
 	// device source input bytes written to log file
-	devSourceBytes map[string]uint64
+	devSourceBytes     map[string]uint64
+	devSourceBytesLock sync.Mutex
```

Comment:
There are locked map definitions which we use elsewhere. Better to use that.

Reply:
Now they are inside